src/Cake.Svn/SvnAliases.Add.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+using Cake.Svn.Add;
+
+namespace Cake.Svn
+{
+    public static partial class SvnAliases
+    {
+        /// <summary>
+        /// Add a directory to Subversion.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directory">The directory.</param>
+        /// <returns>
+        /// <c>true</c> if the command was successful. Otherwise <c>false</c> will be returned.
+        /// </returns>
+        /// <example>
+        /// <para>Add a directory to Subversion.</para>
+        /// <code>
+        /// <![CDATA[ 
+        ///     var directoryAdded = SvnAdd(@"C:\project\src\");
+        ///
+        ///     Verbose("Directory added: {0}", directoryAdded);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Add")]
+        [CakeNamespaceImport("Cake.Svn.Add")]
+        public static bool SvnAdd(this ICakeContext context, DirectoryPath directory)
```

Comment:
This will be ambiguous between `SvnAdd(this ICakeContext, FilePath file)` if the user would write `SvnAdd("foo")`. I'd suggest we rename to `SvnAddDirectory`.